src/SB/Game/zTalkBox.h

```diff
@@ -48,7 +48,8 @@ struct ztalkbox : xBase
     {
         ANSWER_CONTINUE,
         ANSWER_YES,
-        ANSWER_NO
+        ANSWER_NO,
+        ANSWER_3,
```

Comment:
Should probably just call this ANSWER_NUM, though it's somewhat confusing to me that this enum definition in the dwarf doesn't have this value. Possibly sAnswer was just defined as `ztalkbox::ANSWER_NO + 1` but that's also pretty strange.